### PR TITLE
feat: Overwrite address to Connected ICNS Name

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

**Comment by sunnya97** [comment]
Task linked: [CU-862j31wrd Design updates for bridging UI](https://app.clickup.com/t/862j31wrd)

**Comment by sunnya97** [comment]
Task linked: [CU-8669bgure ICNS Frontend - FE Dev](https://app.clickup.com/t/8669bgure)

**Review by jonator — CHANGES_REQUESTED** [review]
Let's not forget the withdraw case as well.

File: packages/web/hooks/use-address-icns-name.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { useStore } from "../stores";
+
+export function useAddressICNSName(bech32Address: string) {
```
[the remark this answers is not in the document]
Comment (JoseRFelix) [reply packages/web/hooks/use-address-icns-name.ts:3]: You're right. Now taking a second look, it's very shallow. Will fix

**Review by jonator — APPROVED** [review]

**Pull request merged**